intro-HPC/gent/macros.tex

```diff
@@ -3,20 +3,22 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
+\macro{\hpc}{UGent-HPC}
 \macro{\hpcname}{Delcatty}
 \macro{\loginnode}{login.hpc.ugent.be}
 \macro{\loginhost}{gligar01.gligar.gent.vsc}
 \macro{\operatingsystem}{Scientifc Linux 6.5}
-\macro{\tutorialdir}{/apps/gent/tutorials}
-\macro{\examplesdir}{/apps/gent/tutorials/Intro-HPC/examples}
+\macro{\tutorialdir}{/apps/antwerpen/tutorials}
+\macro{\examplesdir}{/apps/antwerpen/tutorials/Intro-HPC/examples}
 % Demo user, jobs, nodes, etc
 \macro{\userid}{vsc40000}
 \macro{\homedir}{/user/home/gent/vsc400/vsc40000}
 \macro{\jobid}{123456.master15.delcatty.gent.vsc}
 \macro{\computenode}{node0201.delcatty.gent.vsc}
 \macro{\computenode}{r4e2cn03}
-% Support 
+% Support
 \macro{\hpcinfo}{hpc@ugent.be}
 \macro{\hpcusersml}{hpc-users@lists.ugent.be}
 \macro{\hpcannounceml}{hpc-announce@lists.ugent.be}
+\macro{\operatingsystemSL}{Scientific Linux 6}
```

Comment:
The point of the macro's is that they are all called the same for each institute, but may have a different text in them.
Creating new macro's per institute is not all that useful, also, it should be easy to update the manual for a switch to SL7 by changing the value of this macro.